### DISCUSSION TITLE
Map tilt - how to "untilt"?

### DISCUSSION THREAD

**Discussion by datenbank-projekt** [andreynovikov/trekarta#170] [Q&A] [answered]
Hi,
somehow I tilt the map (put it in 3D) - how can I put it back into 2D?

**Comment by datenbank-projekt** [comment, marked as answer]
Hi,

found it myself: Two fingers on screen and push both up (tilt, switch to 3D) or down (untilt, back to 2D) at the same time.

**Reply by peanutsformonkeys** [reply]
Had the same "challenge", how to get back to 2D modus. Thanks for your answer.

Question for @andreynovikov maybe the tilt angle could also be exposed via a menu item slider, so that this setting is more easily accessible for new users if they don't remember the proper gesture?
